catroid/src/main/java/org/catrobat/catroid/ui/dialogs/SendReportDialog.java

```diff
@@ -58,7 +58,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
 				.setPositiveButton(R.string.crash_dialog_send, new DialogInterface.OnClickListener() {
 					@Override
 					public void onClick(DialogInterface dialog, int id) {
-						Utils.sendCaughtException(getActivity());
+						Utils.sendCaughtException();
```

Comment:
please directly call CrashReporter.sendUnhandledCaughtException(); and remove the utility function.